openpype/settings/defaults/system_settings/applications.json

```diff
@@ -133,7 +133,7 @@
                     "linux": []
                 },
                 "arguments": {
-                    "windows": [],
+                    "windows": ["-U MAXScript {OPENPYPE_ROOT}\\openpype\\hosts\\max\\startup\\startup.ms"],
```

Comment:
If this is always needed, isn't this better to add in a prelaunch hook instead instead of relying on application settings that users tend to adjust?

Reply:
Since we've never found the exact reason, I think it is better to leave it here as the original method with `ADSK_3DSMAX_STARTUPSCRIPTS_ADDON_DIR` is imho more cleaner. I think that with 3dsmax 2023 it less of an issue, but this at least allows to fallback to environment variable way if studio would like to use their own startup script in command line arguments (not sure if you can chain them, probably not)